roles/rhel_subscribe/README.md

```diff
@@ -27,3 +35,9 @@ Custom hostname for the Satellite server (optional).
 Version for the OpenShift Enterprise repositories.
 
 Example: `3.6`
+
+If you want to use user/password to register the instance, it is required to
+configure the `rhsub_user` and `rhsub_pass`.
+
+If you want to use an activation key to register the instance, it is required to
+configure the `rhsub_ak` and `rhsub_orgid`.
```

Comment:
If these are mutually exclusive can you make a note that you should choose one or the other here?

Reply:
I've added a bolded note statement at the beginning of the readme. Thanks!